code/modules/industrial_lift/industrial_lift.dm

```diff
@@ -404,6 +404,9 @@ GLOBAL_LIST_EMPTY(lifts)
 				var/datum/callback/land_slam = new(collided, /mob/living/.proc/tram_slam_land)
 				collided.throw_at(throw_target, 200 * collision_lethality, 4 * collision_lethality, callback = land_slam)
 
+				//Add to the blood counter. Notify the signs that the tram hit someone.
+				SEND_SIGNAL(src, COMSIG_TRAM_COLLISION)
```

Comment:
if the tram hits 10 people in a row, please give them an achievement titled COMBO BREAKER

Reply:
make it play a bowling alley animation on the screen when you hit 10 people at once

Reply:
funny, but we're just trying to get the base of the sign in, add it after when someone has time